config/routes.rb

```diff
@@ -266,6 +266,7 @@
         collection do
           post :clear
           post :dismiss
+          resource :unread, module: :notifications, only: [:show, :update]
```

Comment:
It might make more sense to call the API method "read"? "I have just *read* this notification ID" feels more natural.